resources/public/contracts/src/Offering.sol

```diff
@@ -158,9 +158,9 @@ contract Offering {
     * @param _newOwner address New owner of ENS name
     */
     function transferOwnership(address _newOwner)
-        internal
-        onlyWhenNotEmergencyPaused
-        onlyWithoutNewOwner
+    internal
```

Comment:
It seems the intellij formatter differs from the formatter previously used in the project. Do you know whether there is some standard for this? I highly believe, when Matus makes changes to sol files (using his formatter) he will end up rewriting your (formatting) changes.

Reply:
I haven't seen formatting or style checking set up in the project. For this PR, I'll just remove these formatting changes.